src/inmanta/agent/forking_executor.py

```diff
@@ -287,8 +287,21 @@ async def call(self, context: ExecutorContext) -> typing.Sequence[inmanta.loader
                 logger.info("Failed to load sources: %s", module_source, exc_info=True)
                 failed.append(module_source)
 
+        self.periodic_cache_cleanup_job = asyncio.create_task(self.cleanup_stale_cache_entries(context))
+
         return failed
 
+    async def cleanup_stale_cache_entries(self, context: ExecutorContext) -> None:
```

Comment:
I would expect this code to be in the local executor, as it is the one actually contains it